WickedEngine/wiInput.cpp

```diff
@@ -581,3 +582,56 @@ namespace wiInput
 	}
 
 }
+
+#else
```

Comment:
I'd prefer just extending the functions with #ifdefs for Linux where it's required, instead of reimplementing every function from scratch. (Like I suggested with wiHelper.cpp)